plugins/backstage-opa-backend/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@parsifal-m/plugin-opa-backend",
+  "description": "Then opa backend plugin responsible to interact with the OPA server for validation and authorizations",
```

Comment:
Good catch! 👏 

`The opa backend plugin responsible for interacting with the OPA server for validation and authorisation requests`

Just a NIT!